.github/workflows/release.yaml

```diff
@@ -21,18 +21,15 @@ jobs:
       BIN_NAME: bqls
 
     steps:
-    - uses: actions/checkout@v3
-      with:
-        fetch-depth: 0
-
+    - uses: actions/checkout@v4
```

Comment:
in checkout@v4, default value of fetch-depth is 1, so we can remove this line.
ref: https://github.com/actions/checkout